docs/reduction-in-locks.md

```diff
@@ -0,0 +1,100 @@
+# Reduced backup lock time
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+Percona Server for MySQL implemented `LOCK TABLES FOR BACKUP` to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any `DDL` event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.
```

Comment:
Is this in a Pro build? It should have the Pro Build disclaimer.

Reply:
LOCK  TABLES FOR BACKUP is available in all builds

Reply:
On upstream server, xtrabackup uses LOCK INSTANCE FOR BACKUP. I think we can simplify this.

At some point in docs (early), we can simply say "backup lock" which means
LIFB -> used with upstream/oracle MySQL
LTFB-> used with Percona Server for MySQL

Later, at all places, we can simply say "backup lock"

Reply:
Good points, but I was asking for the Pro Builds disclaimer as the first paragraph if this is part of Pro Builds.